plugins/modules/cloud/misc/proxmox_kvm.py

```diff
@@ -1397,7 +1397,10 @@ def main():
             module.fail_json(msg='VM with name = %s does not exist in cluster' % name)
         vm = proxmox.get_vm(vmid)
         if not name:
-            name = vm['name']
+            try:
+                name = vm['name']
+            except KeyError:
+                name = "_no_name_from_proxmox_"
```

Comment:
1. This can be simplified:
```suggestion
            name = vm.get('name', '_no_name_from_proxmox_')
```
2. Why not use `(no name)` instead of `_no_name_from_proxmox_`? I.e.
```suggestion
            name = vm.get('name', '(no name)')
```